src/bust.py

```diff
@@ -250,12 +256,17 @@ async def ffmpeg_post_hook(e: Exception = None):
             after=ffmpeg_post_hook,
         )
 
-        # Change the name of the bot to that of the currently playing song.
-        # This allows people to quickly see which song is currently playing.
-        new_nick = random_emoji + song_utils.song_format(
+        now_playing_str = song_utils.song_format(
             local_filepath, attachment.filename, submit_message.author.display_name
         )
 
+        # Set now playing title
+        self.now_playing_str = now_playing_str
+
+        # Change the name of the bot to that of the currently playing song.
+        # This allows people to quickly see which song is currently playing.
+        new_nick = random_emoji + now_playing_str
```

Comment:
You can combine these.

```suggestion
        # Set now playing title
        self.now_playing_str = song_utils.song_format(
            local_filepath, attachment.filename, submit_message.author.display_name
        )

        # Change the name of the bot to that of the currently playing song.
        # This allows people to quickly see which song is currently playing.
        new_nick = random_emoji + self.now_playing_str
```